tests/driver_winc1500_netdev/Makefile

```diff
@@ -0,0 +1,50 @@
+APPLICATION = driver_winc1500_netdev
+include ../Makefile.tests_common
+
+BOARD_INSUFFICIENT_MEMORY := msb-430 msb-430h nucleo32-f031 nucleo32-f042 \
+                             nucleo32-l031 nucleo-l053 stm32f0discovery telosb \
+                             weio z1
+
+USEMODULE += winc1500
+USEMODULE += shell
+USEMODULE += shell_commands
+USEMODULE += ps
+
+# Comment out when you don't need GNRC.
+USEMODULE += gnrc_netdev
+USEMODULE += gnrc_netdev_default
+USEMODULE += auto_init_gnrc_netif
+USEMODULE += gnrc_ipv6_router_default
+USEMODULE += gnrc_icmpv6_echo
+
+# Comment this out to disable code in RIOT that does safety checking
+# which is not needed in a production environment but helps in the
+# development process:
+CFLAGS += -DDEVELHELP -g3 -ggdb3 -O0 -Og
+
+# Debugging flags. Uncomment to enable debugging mode
+#CFLAGS += -DLOG_LEVEL=LOG_DEBUG
+
+# set board specific peripheral configurations
+ifneq (,$(filter samr21-xpro,$(BOARD)))
+  # Settings for SAMR21-XPRO Board
+  CFLAGS += -DWINC1500_SPI=\(\SPI_DEV\(1\)\)

```

Comment:
Since some Atmel extensions exists for Samr21-xpro and samd21-xpro boards (and maybe others), I think you can add a winc1500_params.h in the corresponding boards include directories.

Reply:
That sounds better to me too. But in which file you want me add? `boards/<board>/include/board.h` files for each boards or `driver/winc1500/include/winc1500_params.h`?

Reply:
it should go in `boards/sam{d,r,l}21-xpro/include/winc1500_params.h`. It will then be included in place of the driver one. See `boards/nz32-sc151/include/sx127x_params.h` and `drivers/sc127x/include/sx127x_params.h` as example.

Reply:
Thanks for a kind advice :) I looked at the examples. Because they are the same name, I am just wondering if it is alway guaranteed to include `boards/nz32-sc151/include/sx127x_params.h` before `drivers/sc127x/include/sx127x_params.h`?

Reply:
> I am just wondering if it is alway guaranteed to include the boards include first

Normally it is.